src/styles/blocks/card.scss

```diff
@@ -0,0 +1,150 @@
+:root {
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray-color: #616070;
+  --dark-blue-color: #060b35;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+  margin-right: 19px;
+  margin-bottom: 40px;
+  margin-left: 19px;
+}
+
+h2 {
+  all: unset;
+}
+
+.card__product {
+  color: var(--dark-blue-color);
+  width: 166px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: var(--gray-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  margin-right: 84px;
+  margin-left: 16px;
+  margin-bottom: 16px;
+}
+
+.reviews {
```

Comment:
no need to create a bem block for reviews

Reply:
Anastasia Vorobiova said that it is necessary